client/src/components/Leaflet.js

```diff
@@ -201,7 +210,8 @@ Leaflet.propTypes = {
   height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   marginBottom: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
   markers: PropTypes.array,
-  mapId: PropTypes.string // pass this when you have more than one map on a page
+  mapId: PropTypes.string, // pass this when you have more than one map on a page
+  updateMarkers: PropTypes.bool
```

Comment:
Even though `undefined` would work for `bool`, consider adding a default prop of `false` for `updateMarkers`.